index.html

```diff
@@ -67,7 +69,7 @@
 	<body>
 		<ts-root>
 			<ts-header slot="header" class="header" icon="/icon.svg" title="Header">
-				<ts-button>Button</ts-button>
+				<!--<ts-button>Button</ts-button>-->
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') A whitespace is missing after the starting tag at column 8. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3AWhiteSpaceAroundCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a space at column 37. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3AWhiteSpaceAroundCheck)